experimenter/Dockerfile

```diff
@@ -116,14 +115,18 @@ RUN yarn workspace @experimenter/nimbus_ui_new build
 #-------------------------
 FROM python:3.11.1-slim AS deploy
 
+# Create a non-root user with UID 10001
+RUN groupadd -g 10001 appuser && \
+    useradd -r -u 10001 -g appuser appuser
+
 WORKDIR /experimenter
 EXPOSE 7001
 
 # Disable python pyc files
 ENV PYTHONDONTWRITEBYTECODE 1
 
 # Add poetry to path
-ENV PATH "/root/.poetry/bin:${PATH}"
+ENV PATH "/home/appuser/.poetry/bin:${PATH}"
```

Comment:
Is poetry actually required in the final stage?

Reply:
Hrmm good question.  It shouldn't be, let me confirm.